### DISCUSSION TITLE
Possible error mr.splitgradient.

### DISCUSSION THREAD

**Discussion by MJVermeulen** [pulseq/pulseq#58] [Q&A] [answered]
I have the following code: 

GRacq = mr.makeTrapezoid(fe_direction,system,'amplitude', GRacqSamp.amplitude,'FlatTime',readoutTime);

GRparts = mr.splitGradient(GRacq);

But it gives an error for mr.splitGradient(GRacq).
It gives the following error: 

Error using makeExtendedTrapezoid
'maxB1' is not a recognized parameter. For a list of valid name-value pair arguments, see the documentation
for makeExtendedTrapezoid.

Error in mr.makeExtendedTrapezoid (line 32)
parse(parser,channel,varargin{:});

Error in mr.splitGradient (line 42)
    rampup = mr.makeExtendedTrapezoid(ch, opt.system, 'times', times,...

Error in DN_Selpar_All (line 490)
GRparts = mr.splitGradient(GRacq);

It does work on an older version of pulseq. 

If there is an error it could be due to changes in the opts function.

**Comment by m-a-x-i-m-z** [comment]
yes, we indeed broke mr.splitGradient about two weeks ago. I'll fix it ASAP, it's an easy one ;-)

**Comment by m-a-x-i-m-z** [comment, marked as answer]
Should be OK now.

**Comment by MJVermeulen** [comment]
Thanks for the quick fix.
